Backend/src/main/java/ktsnvt/tim1/model/Authority.java

```diff
@@ -0,0 +1,41 @@
+package ktsnvt.tim1.model;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+public class Authority {
```

Comment:
Authority class should implement org.springframework.security.core.GrantedAuthority.